pkg/apis/core/types.go

```diff
@@ -425,25 +425,51 @@ type PersistentVolumeClaimList struct {
 // PersistentVolumeClaimSpec describes the common attributes of storage devices
 // and allows a Source for provider-specific attributes
 type PersistentVolumeClaimSpec struct {
-	// Contains the types of access modes required
+	// Contains the types of access modes the volume should have. Valid access
+	// modes include:
+	// * ReadWriteOnce - can be mounted read/write mode to exactly 1 host.
+	// * ReadOnlyMany - can be mounted in read-only mode to many hosts.
+	// * ReadWriteMany - can be mounted in read/write mode to many hosts.
+	// More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#access-modes-1
```

Comment:
Maybe note that access modes available will depend on what storage plugins are available in the cluster.

Do we want to clarify what it means for multiple access modes to be specified?

Reply:
Or should we just say it's recommended to use 1 access mode only?